### PR TITLE
[7] Develop a Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by elizabethbusola**
## Task: To Develop a Contact Form Page using HTML
- I developed a contact page form using HTML only
contactformpage

### CODE REVIEW

File: submissions/busolaAdedoyin/contactFormPage/contactForm.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <legend>Login Form</legend>
+    <h1>Login Page</h1>
+    <form>
+        <label for="fullname"> Full Name</label><br>
+        <input type="text" name="firstname" id="firstname"><br>
+        <label for="email">Email</label><br>
+        <input type="email" name="email" id="email" placeholder="Enter your email address"><br>
+        <label for="subject">Subject</label><br>
+        <input type="text" name="subject" id="subject"><br>
+        <label for="messageinput">Message</label><br>
```
Comment (ToluAkin) [inline submissions/busolaAdedoyin/contactFormPage/contactForm.html]: Update the label for message in camelCase i.e `messageInput`

File: submissions/busolaAdedoyin/contactFormPage/contactForm.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        <textarea name="messageinput" id="messageinput" cols="30" rows="10"></textarea><br>
```
Comment (ToluAkin) [inline submissions/busolaAdedoyin/contactFormPage/contactForm.html]: Update the name attribute and id as indicated above

File: submissions/busolaAdedoyin/contactFormPage/contactForm.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/busolaAdedoyin/contactFormPage/contactForm.html]: Well done!

File: submissions/busolaAdedoyin/htmlForms/contactForm.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        <input type="email" name="email" id="email" placeholder="Enter your email address"><br>
+        <label for="subject">Subject</label><br>
+        <input type="text" name="subject" id="subject"><br>
+        <label for="messageinput">Message</label><br>
+        <textarea name="messageinput" id="messageinput" cols="30" rows="10"></textarea><br>
+        <button type="submit">Submit</button>
+
+    </form>
+</fieldset>
+
```
Comment (ToluAkin) [inline submissions/busolaAdedoyin/htmlForms/contactForm.html:24]: You can remove this empty line

File: submissions/busolaAdedoyin/contactFormPage/contactForm.html (hunk 2 above)
Comment (elizabethbusola) [reply submissions/busolaAdedoyin/contactFormPage/contactForm.html]: I have made the changes

File: submissions/busolaAdedoyin/htmlForms/contactForm.html (hunk 4 above)
Comment (elizabethbusola) [reply submissions/busolaAdedoyin/htmlForms/contactForm.html:24]: done!

File: submissions/busolaAdedoyin/contactFormPage/contactForm.html (hunk 1 above)
Comment (elizabethbusola) [reply submissions/busolaAdedoyin/contactFormPage/contactForm.html]: I have made the changes

**Pull request merged**